double-press-double-switch.js

```diff
@@ -0,0 +1,79 @@
+// the script show how perform mutliple action with a double classic switch
```

Comment:
Can you provide a bit more description of the use case?

Reply:
description updated with more details